diff --git a/node_modules/@semantic-release/git/lib/prepare.js b/node_modules/@semantic-release/git/lib/prepare.js
index 0e85b00..cc10748 100644
--- a/node_modules/@semantic-release/git/lib/prepare.js
+++ b/node_modules/@semantic-release/git/lib/prepare.js
@@ -27,6 +27,7 @@ module.exports = async (pluginConfig, context) => {
     lastRelease,
     nextRelease,
     logger,
+    envCi,
   } = context;
   const {message, assets} = resolveConfig(pluginConfig, logger);
 
@@ -66,7 +67,8 @@ module.exports = async (pluginConfig, context) => {
         : `chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}`,
       {env, cwd}
     );
-    await push(repositoryUrl, branch.name, {env, cwd});
+    await push(repositoryUrl, envCi.prBranch, {env, cwd});
+    await push(repositoryUrl, envCi.branch, {env, cwd});
     logger.log('Prepared Git release: %s', nextRelease.gitTag);
   }
 };
